iterations.r

```diff
@@ -65,5 +65,33 @@ stations_metadata_df %>%
   theme_classic()
 
 
+### 6: Changing the plot
+stations_metadata_df %>% 
+  filter(latestData > Sys.Date() - days(7)) %>% 
+  # Randomly samples one station from the filtered data and stores it in `sampled_station`
+  sample_n(1) -> sampled_station
+# Extracts the name of the sampled station and stores it in `station_name`
+station_name <- sampled_station$name
+
+sampled_station %$% 
+  vol_qry(
+    id = id,
+    from = to_iso8601(latestData, -4),
+    to = to_iso8601(latestData, 0)
+  ) %>% 
+  GQL(., .url = configs$vegvesen_url) %>%
+  transform_volumes() %>% 
+  # Converts the `from` column to POSIXct format for proper date-time plotting
+  mutate(from = as.POSIXct(from, format="%Y-%m-%dT%H:%M:%S")) %>%
+  ggplot(aes(x=from, y=volume, group = 1)) + 
+  geom_line(aes(color = station_name)) + 
+  theme_classic() +
+  # Labels the color legend as 'Traffic Station'
+  labs(color = 'Traffic Station') + 
+  # Formats the x-axis labels as date-time
+  scale_x_datetime(labels = scales::date_format("%Y-%m-%d %H:%M")) 
```

Comment:
Great plot. Smart to use `scale_x_datetime` to make the x-axis label cleaner.

I would add the station name in the title instead of in a legend, since there is only one group. Legends are typically used when you have multiple groups. 

Also, you could have made the axis titles cleaner. E.g making the first letter uppercase.